.circleci/config.yml

```diff
@@ -39,6 +39,13 @@ jobs:
             pytest --junitxml=test-results/pytest/results-torch.xml tests/torch_*.py
       - store_test_results:
           path: test-results
+      - run:
+          name: Build C API
+          command: |
+            cmake -B build -DBUILD_TESTING=ON -DFAISS_ENABLE_GPU=OFF \
```

Comment:
Let's set `FAISS_ENABLE_C_API=ON` on the first `cmake` call (which should be the only `cmake` call for this job). We can then do `make -C build -j faiss_c` in the last step.